### DISCUSSION TITLE
Data Engine status is orange and RAM and Heap usage are missing

### DISCUSSION THREAD

**Discussion by pwhitney5524** [utmstack/UTMStack#908] [Q&A] [answered]
Hello,

For the past few weeks, when we look at the Data Engine status from the top left of the web interface of our UTMStack intance, the status is orange and we are missing both RAM and Heap usage from that popup when clicked-on.

We tried updating to 10.5.16 but that did not resolve the issue.

Does anyone have ideas?

![UTMStack_ResourceUsage_Issue_1](https://github.com/user-attachments/assets/6b62033b-0d22-48cd-8578-8ea62699ec72)
![UTMStack_ResourceUsage_Issue_2](https://github.com/user-attachments/assets/f96154ca-72af-4e87-ba40-ff8efdc1a67c)

**Comment by c3s4rfred** [comment]
Hi @pwhitney5524 this isn't a bug, those elements were removed in the latest versions.

Best regards

**Comment by pwhitney5524** [comment]
If the Data Engine status is orange, does that mean it is not healthy?

If so, how can we tell what is wrong exactly (where do we need to look)

**Reply by c3s4rfred** [reply]
If the Data Engine status is orange, does that mean it is not healthy? -> Means that the Data Engine is UP but for the moment can be overloaded by queries, available space is low, or resources overloaded, but you don't have to worry about it, RED means down.

If so, how can we tell what is wrong exactly (where do we need to look) -> If you can see logs comming in log explorer view, dashboard showing data, everything is fine, if not, contact the support team.

Best regards

**Reply by osmontero** [reply]
@c3s4rfred this should be somewhere in the documentation, perhaps a monitoring and troubleshooting section might be helpful in understanding these indicators.

**Reply by c3s4rfred** [reply, marked as answer]
Hi, @pwhitney5524 after a review we decided to add a Monitoring and Troubleshooting section in the documentation, to explain those indicators, thanks for the feedback. https://docs.utmstack.com/MonitoringTroubleshooting.html

Best regards
